manifests/kustomize/base/pipeline/metadata-deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metadata-deployment
+  labels:
+    component: metadata-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: metadata-server
+  template:
+    metadata:
+      labels:
+        component: metadata-server
+    spec:
+      containers:
+      - name: container
+        image: gcr.io/kubeflow-images-public/metadata:v0.1.8
+        command: ["./server/server",
+                  "--http_port=8080",
+                  "--mysql_service_host=mysql",
```

Comment:
mysql host/port shouldn't be passed in as parameter.
It should be inferred from metadata pod's env variable 
Example
https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/client_manager.go#L46

Reply:
As per discussion, Keeping this as is for the moment.